packages/unified-latex-ctan/package/listings/provides.ts

```diff
@@ -0,0 +1,24 @@
+import {
+    MacroInfoRecord,
+    EnvInfoRecord,
+} from "@unified-latex/unified-latex-types";
+import {
+    odArgumentParser,
+    ovArgumentParser,
+} from "./libs/command-argument-parser";
+
+export const macros: MacroInfoRecord = {
+    lstset: { signature: "m" },
+    lstinline: { argumentParser: odArgumentParser },
```

Comment:
Does `signature: "o m"` not work?

Reply:
afaik, no, because `m` won't match delim groups e.g. `#code#`. Please correct me if I'm wrong.

Reply:
Would you agree that we can use `o r` as the signature, and implement `r` handling in `gobbleSingleArgument` for both `#verbatim#` and `{verbatim}`?

This may abuse the `r` definition or usage though. In `xparse`, `r` must be presented in the form `r⟨token1 ⟩⟨token2 ⟩`, and (seems) not accepting `{}`.